docs/src/rules/no-implicit-globals.md

```diff
@@ -269,6 +265,12 @@ var global_var = 42;
 
 :::
 
+## Options
+
+This rule has an object option with one option:
+
+* Set `"lexicalBindings"` to `true` if you want this rule to check `const`, `let` and `class` declarations as well.
```

Comment:
I think we should move the ``### `const`, `let` and `class` declarations`` section here, as it's using this option.